src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts

```diff
@@ -138,6 +139,11 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
       this.sortField = this.urlParams.sortField || '';
       this.sortOrder = '1' === this.urlParams.sortOrder ? 1 : -1;
 
+      this.significanceThreshold =
+        this.urlParams.significanceThreshold ||
+        this.DEFAULT_SIGNIFICANCE_THRESHOLD;
+      this.significanceThresholdActive = !!this.urlParams.significanceThreshold;
```

Comment:
Set the significance threshold and display the chiclet if the GCT URL includes a significance parameter when the page is loaded